magento/magento2
# Add regions for Albania,Denmark,Greece,Iceland,Portugal and Sweden

### Description (*)


### Related Pull Requests
https://github.com/magento-commerce/inventory/pull/42


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Install Magento
2. Check table directory_country_region for Albania,Denmark,Greece,Iceland,Portugal and Sweden states/regions.

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31040: Add regions for Albania,Denmark,Greece,Iceland,Portugal and Sweden
